script/release.sh

```diff
@@ -121,12 +158,13 @@ git archive --format=tar --prefix="$project-$version/" "$commit" | xz >"$release
 )
```

Comment:
Nit: It would be nice if the checksum also included the libseccomp artefacts. But I'll do that in a separate PR.